roles/migrationcontroller/templates/mig_rbac.yml.j2

```diff
@@ -71,7 +71,7 @@ rules:
 {% if migration_rsync_privileged %}
   - privileged
 {% else %}
-  - anyuid
+  - rsync-anyuid
```

Comment:
Do you need to this know? if you are setting the user in the SCC?

Reply:
```
  # Create custom SCC for all clusters. For <3.11 scc.users fields will drive the
  # behavior, for others, RBAC will.
  ```
  
 In a hope that one day we will stop supporting 3.10 and less :) 
 
 I can remove it if you feel strongly, I kept it just in case.

Reply:
That is a fair point. I think we should keep it in